src/jarabe/journal/listview.py

```diff
@@ -711,6 +716,17 @@ def __drag_data_get_cb(self, widget, context, selection, info, time):
         model = self.tree_view.get_model()
         model.do_drag_data_get(path, selection)
 
+    def __button_press_event_cb(self, widget, event):
+        try:
+            path = self.tree_view.get_path_at_pos(event.x, event.y)[0]
+        except:
```

Comment:
Oh, both @samdroid-apps and I presumed you were handling an exception.  Now that I know your intent was to protect against None returned, yes, best to split the code into get of path, and extracting first.